parser_test.go

```diff
@@ -54,7 +54,7 @@ func (s *testParserSuite) TestSimple(c *C) {
 		"current_timestamp", "current_user", "database", "databases", "day_hour", "day_microsecond",
 		"day_minute", "day_second", "decimal", "default", "delete", "desc", "describe",
 		"distinct", "distinctRow", "div", "double", "drop", "dual", "else", "enclosed", "escaped",
-		"exists", "explain", "false", "float", "fetch", "for", "force", "foreign", "from",
+		"exists", "explain", "false", "float", "for", "force", "foreign", "from",
```

Comment:
put the `fetch` back

```suggestion
		"exists", "explain", "false", "float", "fetch", "for", "force", "foreign", "from",
```

Reply:
Sorry, I must have made a mistake when merging master.